### ISSUE TITLE
Compiler error watching nested optional value

### ISSUE THREAD

**Issue by acconway** [HerringtonDarkholme/av-ts#58]
Getting `Property 'prop' is optional in type 'Type' but required in type '{ prop: number; }'.` from this code. Is there any way to specify the watch property as optional?
 
```
type Type = {
  prop?: number;
}

class Component extends Vue {
  public member: Type;

  @Watch(['member', 'prop'])
  handler(newValue: number) {
  
  }
}
```

**Comment by HerringtonDarkholme** [comment]
Fixed in 0.7.2

**Comment by acconway** [comment]
👍
